### PR TITLE
Add deb1895 london file

### LINKED ISSUE #54: Create a new file for deb1895 London, UK
- [ ] Create a branch
- [ ] Add Commits
- [ ] Open a PR
- [ ] Have a discussion
- [ ] Seek Approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by dmukherj**
closes #54

### CODE REVIEW

File: deb1895-london.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# London, UK
+
+### Food Recommendations
+- Big Easy
+- Dishoom
+- Pizza Express
+
+### Places to Go
+- Greenwich
+- Fulham Broadway
```
Comment (mrscblowe) [inline deb1895-london.md]: What's playing at the theater?

File: deb1895-london.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# London, UK
+
+### Food Recommendations
```
Comment (achandr2) [inline deb1895-london.md]: Do we have an Indian Restaurant there ?

**Review by mrscblowe — APPROVED** [review]
Thanks for the information.

File: deb1895-london.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# London, UK
+- It's 0 deg C and sunny today.
```
Comment (fabgonzalez) [inline deb1895-london.md:2]: That sounds pretty cold. What is the average temperature for this month?

Comment (dmukherj) [reply deb1895-london.md:2]: around 2-5 deg C - one of the coldest months.

**Pull request merged**